### ISSUE TITLE
As a proposal creator, I need to understand the questions I’m being asked and know how to enter the information I need to provide so that I can submit a new proposal correctly and completely on the first try.

### ISSUE THREAD

**Issue by MelissaBraxton** [USDAForestService/NRM-Grants-Agreements#248]
**Background**
This story covers a bunch of small tweaks to the proposal creation form that we learned about in Sprint Pine Cone Cap user research. This may need to be broken up into smaller issues, but here are the recommendations from the research

 **Project overview**
 - Include cooperator name in project overview
 - Include total amount of funds being requested in project overview
 - Provide link to help text to explain proposed start date fields (may need to break out into new issue)
 -  Remove “will this be covered by FFA?” 
 - Remove “advance allowed” 
 
**When adding partnership details**
 - Remove “new cooperator” question
 ~- Add explanatory text or in context glossary link to help folks understand DUNS#~ (new issue, #323)
 
~**When creating commitments**~
 (moved to new issue #325) 

**We'll know we're done when...**
- [ ] We have implemented refinements in our testing environment

**Tasks**
- [ ] Dev makes updates
- [ ] PO approved

**Comment by tbaxter-18f** [comment]
I'd say we should put prioritize this and have a go at it -- it's a good iterative story showing how research informs further design and development -- and if any of the suggestions seem not ready/doable yet, spin them out as future stories.

Some things, like "Considering allowing the ability to link to multiple workplans." are definitely blocked on the whole Workplan importing spike and work, and the combo box is definitely already in another story, but still, there's a lot of good iteration here.

**Comment by beechnut** [comment]
> Provide link to help text to explain proposed start date fields

@MelissaBraxton Is there pre-written content, or any research that suggests what might need to be written? I'm also noticing that there's some help text for these fields already.
